add marketplace docs (#38)
calimero-is-near/docs: docs/getting_started/market_place.md

---
title: Install applications from Marketplace
sidebar_position: 7
---

> The marketplace allows users to browse and install pre-built smart contracts onto their private shard. By utilizing the marketplace to install contracts in your private shard, you can save time and effort by avoiding the need to write and deploy their own smart contracts. 

Reviewer: ```suggestion
The marketplace allows users to browse and install pre-built smart contracts onto their private shard. By utilizing the marketplace to install contracts in your private shard, you can save time and effort by avoiding the need to write and deploy your own smart contracts. 
```

:::info
Payments and EVM support coming soon
:::

## Install tokens from your marketplace

> To install your tokens from your marketplace, do the following:

Reviewer: ```suggestion
To install smart contracts from your marketplace, do the following:
```

> 1. Navigate to the Calimero [Console](https://app.calimero.network/dashboard).

Author: this is no 2:  In the left navigation menu, Click on the **Marketplace**

> 1. Navigate to the Calimero [Console](https://app.calimero.network/dashboard).

Reviewer: I see that it is, but I'm saying it would make more sense to provide a link directly, rather than guiding the user from the dashboard every time

> 1. Navigate to the Calimero [Console](https://app.calimero.network/dashboard).

Author: this link: https://app.calimero.network/marketplace

2. Click on the **Marketplace** in the left navigation menu.

![](../../static/img/marketplace.png)

4. Click on any of the token you want to install

![](../../static/img/ft_marketplace.png)

5. Click on **Install**

## Viewing  installed applications

> All installed token are viewed from **Applications** which is located in the left navigation menu of the console.

Reviewer: ```suggestion
All installed contracts are viewed from **Applications** which is located in the left navigation menu of the console.
```
